carts/models.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
+
+from products.models import Products, Sizes, Grainding
+from users.models    import User
+from core.models     import TimeStampModel
+
+class Carts(TimeStampModel): 
+    username = models.ForeignKey(User, on_delete      = models.CASCADE)
```

Comment:
외래키면 id값이 데이터로 존재할텐데 username 이라고 작성하신 이유가 있을까요?

`username` -> `user` 로 바꾸는게 적절해 보입니다!
```suggestion
    user = models.ForeignKey(User, on_delete      = models.CASCADE)
```